oio/account/client.py

```diff
@@ -75,7 +72,7 @@ def _maybe_refresh_endpoint(self):
                         raise
                     self.logger.warn(
                             "Failed to refresh account endpoint: %s", exc)
-                except ClientException:
+                except OioException:
```

Comment:
I think you forgot to import OioException here, aswell as remove ClientException import from the top of the file as it is not used anymore.

Reply:
No, that's fine :wink: